### PR TITLE
adds recoil to Discover

### PR DESCRIPTION
**Pull request by ana-yldio**
https://asaphub.atlassian.net/jira/software/c/projects/CRN/boards/8?modal=detail&selectedIssue=CRN-733

### CODE REVIEW

File: apps/frontend/src/discover/Discover.tsx [hunk 1]
```diff
@@ -1,29 +1,22 @@
...
+import { useDiscoverState } from './state';
 
-import { useDiscover } from '../api';
 import Frame from '../structure/Frame';
 
 const loadBody = () => import(/* webpackChunkName: "discover-body" */ './Body');
 const Body = lazy(loadBody);
 loadBody();
 
 const Discover: FC<Record<string, never>> = () => {
```
Comment (marot) [inline apps/frontend/src/discover/Discover.tsx:12]: Is this now using a fallback UI to render the loading state?

File: apps/frontend/src/discover/state.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    return getDiscover(get(authorizationState));
+  },
+});
+
+export const discoverState = atom<DiscoverResponse>({
+  key: 'discoverState',
+  default: fetchDiscoverState,
+});
+
+export const refreshDiscoverState = atom<number>({
```
[the remark this answers is not in the document]
Comment (ana-yldio) [reply apps/frontend/src/discover/state.ts:19]: @marot I think the link is wrong

**Comment by ana-yldio** [comment]
> @ana-yldio did you test the changes in the PR env, or should I have a look at it to be sure?

I test them on the PR env, but can double-check that everything is working as expected:)

File: apps/frontend/src/discover/state.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (marot) [reply apps/frontend/src/discover/state.ts:19]: @ana-yldio ups sorry - updated.

**Comment by marot** [comment]
@ana-yldio I think testing it once should be fine 👍

**Review by niezgoda — APPROVED** [review]

**Pull request closed without merging**